task/pylint/0.2/pylint.yaml

```diff
@@ -41,4 +41,6 @@ spec:
         if [ -n "$(inputs.params.requirements_file)" ] && [ -e "$(inputs.params.requirements_file)" ];then
             python -mpip install --user -r $(inputs.params.requirements_file)
         fi
-        pylint $(inputs.params.args) $(inputs.params.path)
+        pylint $@ $(inputs.params.path)
+      args:
+        - $(inputs.params.args)
```

Comment:
All `inputs.params` should be `params` in `v1beta1` :angel:

Reply:
Arf thanks, i was sure it was the other way around 😺